### PR TITLE
Remove obsolete frameborder attribute

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/880

## QA

- Pull code
- Verify frameborder has been removed

### CODE REVIEW

**Comment by bartaz** [comment]
`frameborder` may be obsolete, but removing it adds border to the `iframe`. If we remove the attribute, we should add CSS to remove the border from the `iframe`. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

File: templates/_layouts/_root.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
...
...);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"///"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
```
Comment (bartaz) [inline templates/_layouts/_root.html:58]: Why is there a change here?
Was usabilla code updated? We shouldn't modify it manually.

**Comment by bartaz** [comment]
@lyubomir-popov Could you rebase this one, Percy is showing some changes between this branch and master from previously merged PR.

**Comment by lyubomir-popov** [comment]
@bartaz rebased.

**Review by bartaz — APPROVED** [review]
LGTM

**Pull request merged**